index.js

```diff
@@ -19,16 +18,7 @@ function sizeof(object) {
       else {
         var bytes = 0;
         _.forOwn(object, function (value, key) {
-          bytes += sizeof(key);
-          try {
-            bytes += sizeof(value);
-          } catch (ex) {
-            if(ex instanceof RangeError) {
-              // circular reference detected, final result might be incorrect
```

Comment:
What about circular dependencies? Maybe use ES6 Set to store all primitive objects found, and then iterate over this set and calculate sizes ?